### DISCUSSION TITLE
How to accelerate IPSec tunnel using a “relay” server?

### DISCUSSION THREAD

**Discussion by charliex2** [strongswan/strongswan#1246] [Q&A] [answered]
I establised a ipsec tunnel using StrongSwan between two counties far away, and the network speed is slow.
Is it possible to deploy a “relay” server between these two counties to accelerate the tunnel network?
Is there any other methods to optimize the tunnel network?

**Comment by Thermi** [comment, marked as answer]
Not exactly accelerate but you could influence the routing by just routing between them over a third IPsec server.
I don't know of any public/open-source WAN optimization you could be using, sorry.
